eks-hybrid-iam-ra-setup.sh

```diff
@@ -51,6 +69,20 @@ cat << EOF > "${SECRETS_DIR}/node.key"
 ${NODE_KEY_DATA}
 EOF
 
+if [ "${ENABLE_CREDENTIALS_FILE}" = "true" ]; then
+    EKS_HYBRID_SHARED_CREDENTIALS_FILE="/root/.aws/eks-hybrid/credentials"
+    SIGNING_HELPER_UPDATE_COMMAND="$(get_aws_signing_helper_update_command ${SECRETS_DIR}/config)"
+    export EKS_HYBRID_SHARED_CREDENTIALS_FILE SIGNING_HELPER_UPDATE_COMMAND
+    # shellcheck disable=SC2016 # we want to replace the variables verbatim
+    envsubst '$EKS_HYBRID_SHARED_CREDENTIALS_FILE:$SIGNING_HELPER_UPDATE_COMMAND' \
+        < "${SIGNING_HELPER_SERVICE_TEMPLATE_PATH}" \
+        > "${SIGNING_HELPER_SERVICE_PATH}"
+    chroot ${HOST_ROOTFS} systemctl enable ${SIGNING_HELPER_SERVICE} --no-reload
+    hybrid_nodes_pod_identity_aws_dir="${HOST_ROOTFS}/var/eks-hybrid/.aws"
+    mkdir -p "$(dirname "${hybrid_nodes_pod_identity_aws_dir}")"
```

Comment:
nit: `dirname` is redundant

Reply:
It was required if I didn't have the `-n` flag to `ln` in the line below, because otherwise it was treating the symlink as a directory and creating the actual symlink one level below, i.e. `/var/eks-hybrid/.aws/.aws`. I added the `-n` flag and removed the `dirname`.